src/Microsoft.DocAsCode.Metadata.ManagedReference/Visitors/SymbolVisitorAdapter.cs

```diff
@@ -608,6 +612,25 @@ where FilterVisitor.CanVisitApi(t)
             }
         }
 
+        private void GenerateExtensionMethods(INamedTypeSymbol symbol, MetadataItem item)
+        {
+            var extensions = new List<string>();
+            foreach (var e in _extensionMethods)
+            {
+                var reduced = e.ReduceExtensionMethod(symbol);
+                if ((object)reduced != null)
```

Comment:
follow Roslyn's way

---
In reply to: [65018971](https://github.com/dotnet/docfx/pull/339#discussion_r65018971) [](ancestors = 65018971)